OpenRA.Mods.Common/Activities/Turn.cs

```diff
@@ -40,5 +43,10 @@ public override Activity Tick(Actor self)
 
 			return this;
 		}
+
+		public override void DrawLines(Actor self, List<Target> currentTargets)
+		{
+			self.SetTargetLines(currentTargets, Color.Red);
```

Comment:
Why is the color red and why do we draw a line in a turning activity that doesn't change the actors position?